### PR TITLE
Replaces armoury lockers with Gun and Ammo Vendors.

### PR DESCRIPTION
**Pull request by X0-11**
Heavy weapons and normal weapons have been split into 2 vendors.

Credit to WarlordToby for the sprites.
No alternate sprites for heavy and normal weapon racks though.

### CODE REVIEW

**Comment by bloxgate** [comment]
Is this ready for review and merge? Or is there still more coming?

**Comment by X0-11** [comment]
Besides alternate sprites for heavy weapons vendors, it's ready for merge.
The sprites can always be added later

**Comment by bloxgate** [comment]
Great I may have to add this soon. We've had quite a bit of lag from the armory lockers being opened.

**Comment by bloxgate** [comment]
Going to give this a quick once over, then merge it and open an issue for updated sprites if everything is good.

**Review by bloxgate — CHANGES_REQUESTED** [review]
Mind moving the vending machines into the halo module? It'll help us if we ever merge any future bay updates to vending machines.

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -964,5 +964,4 @@
 					/obj/item/weapon/cell/high = 12, /obj/item/device/assembly/prox_sensor = 3,/obj/item/device/assembly/signaler = 3,/obj/item/device/healthanalyzer = 3,
 					/obj/item/weapon/scalpel = 2,/obj/item/weapon/circular_saw = 2,/obj/item/weapon/tank/anesthetic = 2,/obj/item/clothing/mask/breath/medical = 5,
 					/obj/item/weapon/screwdriver = 5,/obj/item/weapon/crowbar = 5)
-	//everything after the power cell had no amounts, I improvised.  -Sayu
-
+	//everything after the power cell had no amounts, I improvised.  -Sayu
```
Comment (caelaislinn) [inline code/game/machinery/vending.dm]: Unnecessary line change

**Review by caelaislinn — APPROVED** [review]
I'm not a fan of gun vendors from a gameplay perspective but we don't necessarily have to use them. LGTM

**Comment by caelaislinn** [comment]
Map conflict with #260 needs to be resolved now that it's merged

**Comment by X0-11** [comment]
I'll resolve the map conflict once I get home.
Online editor is annoying. Sorry about the extra two commits.

**Comment by X0-11** [comment]
Not too sure how I'm meant to fix the conflict. I've remapped it using the map from alpha

**Comment by X0-11** [comment]
I've borked something. Will open another once I get it to work

**Pull request closed without merging**